### PR TITLE
Update flaky-tests.md

### PR DESCRIPTION
**Pull request by aojea**
- Zero-flake policy
- Update resources

### CODE REVIEW

File: contributors/devel/sig-testing/flaky-tests.md [hunk 1]
```diff
@@ -10,8 +10,14 @@ writing our tests defensively. When flakes are identified, we should prioritize
 addressing them, either by fixing them or quarantining them off the critical
 path.
 
-For more information about deflaking Kubernetes tests, watch @liggitt's
-[presentation from Kubernetes SIG Testing - 2020-08-25](https://www.youtube.com/watch?v=Ewp8LNY_qTg).
+The project has a "zero-flake" policy, jobs must not automatically retry on test failures,
+this was effectivily implemented and announced in 2019,
+[No more ginkgo.flakeAttempts=2 for e2e tests as of 2019-12-13}(https://groups.google.com/g/kubernetes-dev/c/NNmEGUsJObg/m/dmI2mVc_AAAJ)
```
Comment (xmudrii) [inline contributors/devel/sig-testing/flaky-tests.md:15]: Markdown typo:
```suggestion
[No more ginkgo.flakeAttempts=2 for e2e tests as of 2019-12-13](https://groups.google.com/g/kubernetes-dev/c/NNmEGUsJObg/m/dmI2mVc_AAAJ)
```

**Review by xmudrii — COMMENTED** [review]
NB +1

**Review by sftim — COMMENTED** [review]
+0 (no objection)

Optional feedback on the wording

**Comment by enj** [comment]
LGTM

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/hold

**Comment by mrbobbytables** [comment]
I think this is safe to unhold at this point? @aojea @BenTheElder

**Comment by aojea** [comment]
> I think this is safe to unhold at this point? @aojea @BenTheElder

/hold cancel

Thanks

**Pull request merged**